calendar-api/README.md

```
@@ -0,0 +1,21 @@
+# calendar-api
+
+This is a small service that exposes all calendar events, combining
+data from Confluence with older events.
+
+Events are exposed as JSON (consumer in the frontend) and as iCalendar file.
+
+https://foreningenbs.no/calendar-api/events
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)